src/components/patient/patientCard/patientInfo/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import { Tag } from "@blueprintjs/core";
+import "./style.css";
+
+interface Props {
+  infoType: string;
```

Comment:
do you need to prefix these attributes by `info` ? The component is already called patientInfo, I think it would make sense to have the props `type` and `content` (or `value`)